teams/passguard.md

```diff
@@ -0,0 +1,10 @@
+## PASSGUARD 
+
+## Developers 
+
+* [Mateo Guzman] (https://github.com/mateoguzv)
```

Comment:
Links need to have no space between the `](` so that the square brackets are followed immediately by the parentheses